src/tests/topp/THIRDPARTY/SiriusAdapter_1_output.mzTab

```diff
@@ -1,60 +1,4 @@
-SML	null	C15H28O4	null	null	null	295.18707248574799	null	null	418.393999999979997	null	null	null	null	null	null	6.0	6.0	0.0	null	[M + Na]+	C15H28O4	1	1	0.0	nan	nan	-3.074627675571169	43	2199003	null	sample=1 period=1 cycle=2199 experiment=3
-SML	null	C13H28N4O	null	null	null	295.18707248574799	null	null	418.393999999979997	null	null	null	null	null	null	1.0	1.0	0.0	null	[M + K]+	C13H28N4O	2	1	0.0	nan	nan	-8.12071586440215	43	2199003	null	sample=1 period=1 cycle=2199 experiment=3
-SML	null	C12H26N2O6	null	null	null	295.18707248574799	null	null	418.393999999979997	null	null	null	null	null	null	1.0	1.0	0.0	null	[M + H]+	C12H26N2O6	3	1	0.0	nan	nan	2.403498403688778	43	2199003	null	sample=1 period=1 cycle=2199 experiment=3
-SML	null	C16H24N4	null	null	null	295.18707248574799	null	null	418.393999999979997	null	null	null	null	null	null	1.0	1.0	0.0	null	[M + Na]+	C16H24N4	4	1	0.0	nan	nan	-7.605286788519772	43	2199003	null	sample=1 period=1 cycle=2199 experiment=3
-SML	null	C11H24N6O2	null	null	null	295.18707248574799	null	null	418.393999999979997	null	null	null	null	null	null	1.0	1.0	0.0	null	[M + Na]+	C11H24N6O2	5	1	0.0	nan	nan	6.022545779223315	43	2199003	null	sample=1 period=1 cycle=2199 experiment=3
-SML	null	C12H19NO7	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.829331145050649	0.829331145050649	0.0	null	[M + H]+	C12H19NO7	1	1	0.0	1.979288701546372	1.979288701546372	1.979288701546372	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C13H15N5O3	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.767458571705934	0.767458571705934	0.0	null	[M + H]+	C13H15N5O3	2	1	0.0	-2.6304367531418	2.6304367531418	-2.6304367531418	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C11H17N5O3	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.440145632853194	0.440145632853194	0.0	null	[M + Na]+	C11H17N5O3	3	1	0.0	5.661493518486198	5.661493518486198	5.661493518486198	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C9H17N9	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.29900758475931	0.29900758475931	0.0	null	[M + K]+	C9H17N9	4	1	0.0	0.527344029052501	0.527344029052501	0.527344029052501	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
-SML	null	C16H17N3O	null	null	null	290.124002639250989	null	null	420.444000000000017	null	null	null	null	null	null	0.11321656639466	0.11321656639466	0.0	null	[M + Na]+	C16H17N3O	5	1	0.0	-8.204163798395776	8.204163798395776	-8.204163798395776	49	2204006	null	sample=1 period=1 cycle=2204 experiment=6
+MTD	description	null
```

Comment:
That looks very bad unfortunately

Reply:
were there changes in their folder structure/ format such that we cannot read it anymore?

Reply:
The reason for the empty files was the missing `write-summaries` flag for the Sirius 5.